ppr-ui/src/utils/mhr-api-helper.ts

```diff
@@ -579,3 +585,61 @@ export async function deleteMhrDraft (draftID: string): Promise<ErrorIF> {
 export function delayActions (milliseconds: number): Promise<any> {
   return new Promise(resolve => setTimeout(resolve, milliseconds))
 }
+
+const UIFilterToApiFilter = {
+  endDate: 'createDateTime',
+  folNum: 'clientReferenceId',
+  regBy: 'username',
+  regNum: 'mhrNumber',
+  regType: 'registrationType',
+  startDate: 'createDateTime',
+  status: 'statusType',
+  regParty: 'submittingName'
+}
+
+// add sorting params for registration history/draft api calls
+function addSortParams (url: string, sortOptions: RegistrationSortIF): string {
+  const sortKeys = Object.keys(sortOptions)
+  // add all set filters as params to the call
+  for (const i in sortKeys) {
+    // convert to api expected value (too tied in with header logic to change earlier)
+    if (sortOptions[sortKeys[i]] === 'createDateTime') {
+      // sortKeys[i] === orderBy (only case this will happen)
+      sortOptions[sortKeys[i]] = 'startDateTime'
+    }
+    // add timestamp onto datetime param values
+    if (sortOptions[sortKeys[i]] && ['createDateTime'].includes(UIFilterToApiFilter[sortKeys[i]])) {
+      // ensure its not already converted
+      if (sortOptions[sortKeys[i]].length < 11) {
+        sortOptions[sortKeys[i]] = sortOptions[sortKeys[i]]
+        // convert to local date object
+        const d = new Date(`${sortOptions[sortKeys[i]]}T00:00:00`)
+        // get the offset from utc in hours
+        let offset = `${d.getTimezoneOffset() / 60}`
```

Comment:
Maybe this can go to a **util** file to keep **mhr-api-helper** cleaner?

Reply:
Good idea.  Added to utilities in utils.